star_tides/api/decorators/test_login_required.py

```diff
@@ -0,0 +1,42 @@
+''' star_tides.api.decorators.test_login_required
+'''
+# Disabled because pytest adds fixtures by having them specified as arguments.
+# This overloads the import which isn't allowed in google's pylintrc.
+# pragma pylint: disable=W0621,W0613,W0611,W0703
+
+import base64
+from star_tides.tests.utils import response_to_dict
+from star_tides.tests.conftest import client
+from star_tides.tests.fixtures.basic_user import basic_user
+
+
+def test_login_required_decorator(client, basic_user):
```

Comment:
Question: how do we "deactivate" (or better, fake) auth for tests?

Reply:
An example of the auth is in that test:
```py
    user = basic_user()
    email = user['email']

    encoded_email_and_password = \
        base64.b64encode(bytes(f'{email}:password', 'utf-8'))

    header = {'Authorization': 'Basic {}'.format(
        encoded_email_and_password.decode('utf-8'))}

    response = client.post('/auth/login',
                           headers=header)
```

Reply:
Because it's kind of encumbersome to do this every time, so I might make a helper method that we pass a route, email, and body for.